scan-payment/src/main/java/com/getbouncer/scan/payment/ml/ExpiryDetect.kt

```diff
@@ -0,0 +1,136 @@
+package com.getbouncer.scan.payment.ml
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.util.Size
+import com.getbouncer.scan.framework.Loader
+import com.getbouncer.scan.framework.ResourceLoader
+import com.getbouncer.scan.framework.ml.TFLAnalyzerFactory
+import com.getbouncer.scan.framework.ml.TensorFlowLiteAnalyzer
+import com.getbouncer.scan.framework.util.indexOfMax
+import com.getbouncer.scan.payment.R
+import com.getbouncer.scan.payment.hasOpenGl31
+import com.getbouncer.scan.payment.scale
+import com.getbouncer.scan.payment.toRGBByteBuffer
+import org.tensorflow.lite.Interpreter
+import java.io.FileNotFoundException
+import java.nio.ByteBuffer
+
+private val TRAINED_IMAGE_SIZE = Size(80, 36)
+
+private const val NUM_CLASS = 11
+private const val NUM_PREDICTIONS = 17
+private const val BACKGROUND_CLASS = 10
+
+val ASPECT_RATIO = TRAINED_IMAGE_SIZE.height.toFloat() / TRAINED_IMAGE_SIZE.width.toFloat()
```

Comment:
this should be private, any only accessed by this class. This class should be in charge of cropping / manipulating the input image as needed.